src/helpers/makeUpperCase.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generic to lower case
+ *
+ * @function
+ *
+ * @param {string} word
+ *
+ * @return {Function} call to turn upper case to lower case
+ */
+const makeUpperCase = (word) => {
```

Comment:
Make uppercase? I think you mean Capitalize. This is uppercased, `UPPERCASE`, this is lowercased, `lowercase` and this is capitalized, `Capitalize`. Please correct it

Reply:
Done.